tests/integration/helpers/helpers.py

```diff
@@ -229,16 +231,16 @@ def wait_for_relation_removed_between(
     try:
         for attempt in Retrying(stop=stop_after_delay(3 * 60), wait=wait_fixed(3)):
             with attempt:
-                if relation_exited(ops_test, endpoint_one, endpoint_two):
-                    break
+                if not relation_exited(ops_test, endpoint_one, endpoint_two):
+                    raise Exception("Relation not exited")
```

Comment:
You can just assert `relation_exited`

Reply:
addressed in 79f0de8362d1cb5f3d00125a778b4c39e1d5e91d